serveradmin/common/management/commands/createdefaultuser.py

```diff
@@ -0,0 +1,31 @@
+"""Serveradmin
+
+When upgraded to Django >= 3.x we can use DJANG_SUPERUSER_* environment
+variables together with the createsuperuser command and the --non-interactive
+to do this.
+
+Copyright (c) 2021 InnoGames GmbH
+"""
+from os import environ
+
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Create default user based on environment variables'
+
+    def handle(self, *args, **options):
+        username = environ.get('DJANGO_SUPERUSER_USERNAME')
+        password = environ.get('DJANGO_SUPERUSER_PASSWORD')
+
+        if not all([username, password]):
+            raise CommandError('Missing at least one DJANG_SUPERUSER_* env')
+
+        if not User.objects.filter(username=username).exists():
```

Comment:
If the user already exists, should that command maybe update the existing user to make sure that the user is a staff and superuser?

Reply:
No it should not for sake of security. The commands intention is to be used to setup an development environment and not override or update existing users.